api/osrd_infra/schemas/infra.py

```diff
@@ -75,17 +108,29 @@ class LoadingGaugeType(str, Enum):
 
 
 class DirectionalTrackRange(BaseModel):
-    track: ObjectReference
-    begin: float
-    end: float
-    direction: Direction
+    """
+    This class is used to define the path of the route used by the train.
+    """
+
+    track: ObjectReference = Field(description="Identifier and type of the track")
```

Comment:
```suggestion
    track: ObjectReference = Field(description="Identifier of the track")
```

Tracks don't really have a type. The type in `ObjectReference` is always `"TrackSection"` here, it just adds a kind of typing to references in general

This also applies to every mention of `Identifier and type` in the diff